appeals/web/src/server/appeals/appeal-documents/appeal-documents.controller.js

```diff
@@ -43,6 +52,8 @@ export const renderDocumentUpload = async (request, response, backButtonUrl, nex
 		blobStorageContainer: config.blobStorageDefaultContainer,
 		multiple: !documentId,
 		documentStage: documentStage,
+		serviceName: documentName,
```

Comment:
I think these fall back to default values if undefined, but just making sure

Reply:
They are undefined but ignored by the nunjucks template.